general/g.message/g.message.html

```diff
@@ -47,7 +47,7 @@ <h2>NOTES</h2>
 
 <h3>Usage in Python scripts</h3>
 
-<a href="https://grass.osgeo.org/grass80/manuals/libpython/">GRASS
+<a href="https://grass.osgeo.org/grass81/manuals/libpython/">GRASS
```

Comment:
We have grass-stable, so why not grass-development, grass-preview (following naming on the website), or grass-main (following branch name)? (It makes sense to me to merge this PR without/before implementing this suggestion.)

Reply:
Indeed we also have 
https://grass.osgeo.org/grass-devel/manuals/

At time it doesn't point to anything useful but once this PR is merged we will have grass81 on the server shortly.

So I'll update this PR accordingly.

Reply:
Done in 5af69faf5890248607d33620a230ffc864e72434

Reply:
Great! Thanks.